internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
```

Comment:
Should we name this one Test-ComputerConnection?
- it does test both ping and CIM / WMI
- it's similar to Test-SqlConnection but at host level
- this doesn't raise te expectation of having a connection object